templates/emails/payment.creditcard.expiring.hbs

```diff
@@ -0,0 +1,45 @@
+Subject: Your credit card is about to expire
+
+<style>
+  code {
+    display: inline-block;
+    border: 1px solid #ccc;
+    background: #ddd;
+    padding: 1px 3px;
+    margin: 0px 1px;
+  }
+</style>
+
+{{> header}}
+
+{{> toplogo}}
+
+{{#if firstName}}
+<p>Dear {{firstName}},</p>
+{{else}}
+<p>Hi!</p>
+{{/if}}
+
+<p>
+  A credit card used for your recurring subscriptions will expire at the end of this month.
```

Comment:
As a reminder, final copy needs to be provided.

Reply:
Provided by design or would you like me to come up with something?

Reply:
Note: will come up with something together today.